authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -24,6 +24,7 @@ include::modules/manually-maintained-credentials-upgrade.adoc[leveloffset=+1]
 [id="additional-resources_cco-mode-manual"]
 == Additional resources
 
+* xref:../../installing/installing_alibaba/manually-creating-alibaba-ram.adoc#manually-creating-alibaba-ram[Manually creating RAM resources for Alibaba Cloud]
```

Comment:
Right before the Additional resources ID, we need to add:
[role="_additional-resources"]
per the updated guidance https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections